server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTreeBelowSGCachedImpl.java

```diff
@@ -1065,4 +1011,39 @@ public void updateMNode(IMNode node) throws MetadataException {
 
   // endregion
 
+  // region Interfaces for schema reader
+  public ISchemaReader<INodeSchemaInfo> getNodeReader(IShowNodesPlan showNodesPlan)
```

Comment:
Since the MTreeBelowSGCachedImpl and MTreeBelowSGMemoryImpl are used in different SchemaRegionImpl and there's no scenarios that they serve the same invoker, I think we can remove the interface ```IMTreeBelowSG``` in future work.